webapp/src/components/Analysis/UtterancesTable.tsx

```diff
@@ -376,7 +376,7 @@ const UtterancesTable: React.FC<Props> = ({
     <Box className={classes.gridContainer}>
       <div className={classes.gridHeaderActions}>
         <Description
-          text="View insights on individual utterances with details results and annotations."
+          text="Explore the utterances and propose actions. Click on a row for the utterance details."
```

Comment:
Let's add a verb in the second sentence. Ideas:
```suggestion
          text="Explore the utterances and propose actions. Click on a row to inspect the utterance details."
```
```suggestion
          text="Explore the utterances and propose actions. Click on a row to get to the utterance details."
```
```suggestion
          text="Explore the utterances and propose actions. Click on a row to dig into the utterance details."
```